easybuild/easyconfigs/m/Mesa/Mesa-11.0.2-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'ConfigureMake'
+
+name = 'Mesa'
+version = '11.0.2'
+
+homepage = 'http://www.mesa3d.org/'
+description = """Mesa is an open-source implementation of the OpenGL specification -
+ a system for rendering interactive 3D graphics."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'optarch': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'ftp://ftp.freedesktop.org/pub/mesa/%(version)s',
+    'ftp://ftp.freedesktop.org/pub/mesa/older-versions/%(version_major)s.x/%(version)s',
+]
+
+pythonver = '2.7.10'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+versionsuffix = '-%s-%s' % ('Python', pythonver)
+
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('Bison', '3.0.4'),
+    ('Autotools', '20150215', '', ('GNU', '4.9.3-2.25')),
+    ('pkg-config', '0.29'),
+    ('glproto', '1.4.17'),
+    ('kbproto', '1.0.7'),
+    ('xextproto', '7.3.0'),
+    ('xproto', '7.0.28'),
+    ('makedepend', '1.0.5'),
```

Comment:
OK, so Autotools load libtool, which has M4 as a runtime dep, that's where the conflict comes.

But, that also means there's no need to include it as a direct dep here, since M4 will be loaded as a build dep when building Mesa.